mapie/quantile_regression.py

```diff
@@ -363,16 +369,41 @@ def fit(  # type: ignore
         alpha = self._check_alpha(self.alpha)
         self.cv = self._check_cv(cast(str, self.cv))
         X, y = indexable(X, y)
+        if X_calib is None or y_calib is None:
+            if sample_weight is not None:
```

Comment:
I would start with `if sample_weight is None`

Reply:
done